### PR TITLE
[C-387 C-434] Set button text to be static white on hover and add button icon transition

### PR DESCRIPTION
**Pull request by Kyle-Shanks**
[C-387 C-434]

* Set button text to be darkmode-static-white so that it will show as white in dark mode
* Update icon transition to make the button hover transition nicer

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,6 +9,7 @@ example/node_modules
 build
 dist
 .rpt2_cache
+storybook-static
```
Comment (nicoback2) [inline .gitignore:12]: intentional?

Comment (Kyle-Shanks) [reply .gitignore:12]: added bc this was generated when i built, but doesn't need to be in this PR 👍

File: src/components/Button/Button.module.css [hunk 2]
```diff
@@ -168,12 +169,12 @@
 }
 
 .secondary.includeHoverAnimations:hover .textLabel {
-  color: var(--white);
+  color: var(--darkmode-static-white);
```
Comment (sliptype) [inline src/components/Button/Button.module.css:172]: I don't see `--darkmode-static-white` in colors.css, am I missing it or does it only exist in audius-client?

Also could you explain the difference between `--darkmode-static-white` and `--static-white` to me? I am confusion

**Review by sliptype — COMMENTED** [review]
Looks good! But I am a lil confused on our colors

**Pull request merged**